### ISSUE TITLE
As a Project Coordinator, I want to the optional tasks in the list to be initially hidden, so that I am not overwhelmed by the amount of forms

### ISSUE THREAD

**Issue by matthieu-foucault** [bcgov/cas-cif#371]
#### Description:

Brief description of the work involved, including any user research context or key decisions.

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Notes:

-

**Comment by pbastia** [comment]
Maybe this requires a bit of UX design: how do we hide those tasks? What is the interaction to show/hide them? And do they need to be hidden depending on how many there are, or can we achieve the same result (user not overwhelmed) with some clever symbology?

**Comment by matthieu-foucault** [comment]
Oh there is a design, it's just that triangle to the right of the project details heading. I just hadn't had the chance to add it to the issue. I do have some questions for @suhafa though: 

- do we rotate it when it's collapsed? 90 degrees or 180 degrees?
- do we expand/collapse when clicking the entire list item (heading + icon)? 

![image](https://user-images.githubusercontent.com/2684022/160507677-b1490e24-4052-4bdd-85e6-e5384f7132fe.png)

**Comment by suhafa** [comment]
@matthieu-foucault @pbastia 
- When the list is collapsed, the icon (the caret/triangle in this case) would be pointing upwards. When opened, it would rotate 180 degrees. 
- Having the heading + icon to be clickable is a great idea, would increase surface area for user to click - making it easier. (referenced https://www.nngroup.com/articles/accordion-icons/ to ensure this is a good idea)

Another point or a good-to-consider (perhaps a question for development): Let us assume a project is in its infancy, would it make sense for only relevant items from the task list to be open by default? For the rest of the items that would be irrelevant at that time (say Milestone Reporting), could we have it collapsed and greyed out? (although the user can click on it and navigate if need be to give them additional flexibility) Perhaps this ties in with our questions about the approval process and roping it in the UX/UI - as the different stages get approval, more items would appear open and/or only relevant items would be open. Another instance that comes to mind is if there is a need for approval, we could have items that need reviewing open by default so that the team can directly navigate to the task that needs their attention. Just a note, we could explore this with a task of its own.

**Comment by pbastia** [comment]
@suhafa I think the initial goal was to leave things as open as possible, to get towards a usable system as early as we can, so maybe we don't have to worry about approval process and project stage quite yet.

I'm wondering how to handle the "resume creation" when some of the optional forms have been started? Do we hide the ones that haven't been started yet? How does the UI make it clear that there are some hidden forms and some shown? Or do we simply just show all the available forms in that case?

**Comment by matthieu-foucault** [comment]
Good point. I think that if any of the forms has a status other than `Not Started`, then the section should be expanded by default

**Comment by matthieu-foucault** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/371?planning-poker) with ZenHub @BCerki @dleard @gurjmatharu @pbastia @Sepehr-Sobhani
